### DISCUSSION TITLE
1.0.0 Features Roadmap

### DISCUSSION THREAD

**Discussion by Baroshem** [nuxt-modules/algolia#55] [Ideas]
I am planning to release a big new version of the module with several changes to the module itself. Currently these are the features that I added to the roadmap:

- #47 
- #53 
- #54 
- #46 
- #35

**Comment by chuckmeyer** [comment]
Add support for click/conversion event capture for metrics and recommendation mode training.

https://www.algolia.com/doc/guides/building-search-ui/going-further/send-insights-events/vue/
